src/components/shared/Terminal.tsx

```diff
@@ -64,18 +66,25 @@ function Terminal(prop: {
   }
 
   function executeTouch(path: string, directory = false): string[] {
+    // Make a copy to avoid mutating the original (it's a state variable)
     const fileSystemCopy = _.cloneDeep(prop.fileSystem);
```

Comment:
Refactor - could we rename this to rootCopy so the naming is less confusing? Thanks 🙂!

Reply:
Yea, this is clearer. A future backlog task would be to refactor and change all instances of fileSystem to be simply root (including changing the name in state).